### ISSUE TITLE
[Bug]: Updating/submitting ratings fails with an infinite loading animation

### ISSUE THREAD

**Issue by coglebed** [nsunami/postreview-app#87]
### URL of the Page

https://postreview-app.herokuapp.com/articles/2fe810b8-9d30-42a3-9c3e-682e3213e7e8

### What happened?

Submitting review for two papers below (only possible with one click due to 400 error thing) leads to an infinite loading.

https://postreview-app.herokuapp.com/articles/2fe810b8-9d30-42a3-9c3e-682e3213e7e8
https://postreview-app.herokuapp.com/articles/b7c3c3ee-26ca-4d34-a4cd-d123acc65d3f

### Steps to reproduce

1. click on the paper
2. click once on any of the ratings
3. click submit

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nsunami** [comment]
I think the cause of this issue can be (a) we are not catching empty/partial ratings, or (b) we are not successfully updating the ratings. Will investigate.

**Comment by coglebed** [comment]
Do we want to let users upload incomplete ratings in the first place? I'm thinking probably yes - especially if we plan to introduce other questions in the future (this would mean that old reviews will get an empty field retrospectively). Perhaps the better solution here would be to allow incomplete ratings and start thinking about how those should be visually displayed? (e.g. grayed out star)

**Comment by coglebed** [comment]
@nsunami will make the mock-up. Do you want me to create a separate issue or continue through #94?

**Comment by coglebed** [comment]
How about [this mock-up](https://whimsical.com/home-page-XX3QdWKnjFvf755mCRr4CD@2Ux7TurymPBxHdS1jC2H)? The big start for the paper card and the small stars for the rating card just being gray is a little confusing because it might mean that there are no ratings or that all the ratings so far have been zero. To distinguish the two I've also added a "N/A" label in the center of the big start, while I removed the stars entirely in the rating card case and replaced them with "N/A". Curious to hear your thoughts on whether N/A is the appropriate label in these cases and whether you think removing small starts is the way to go to make it the least confusing.
